wire-ios-data-model/Source/Model/Message/ZMOTRMessage+UpdateEvent.swift

```diff
@@ -52,7 +52,8 @@ extension ZMOTRMessage {
                 return nil
         }
         zmLog.debug("Processing:\n\(message)")
-
+        let logAttributes: LogAttributes = [LogAttributesKey.eventId.rawValue: updateEvent.safeUUID, LogAttributesKey.nonce.rawValue: updateEvent.messageNonce, LogAttributesKey.messageType.rawValue: updateEvent.safeType]
```

Comment:
line break formatting would help me to read:
```suggestion
        let logAttributes: LogAttributes = [
            LogAttributesKey.eventId.rawValue: updateEvent.safeUUID,
            LogAttributesKey.nonce.rawValue: updateEvent.messageNonce,
            LogAttributesKey.messageType.rawValue: updateEvent.safeType
        ]
```